query_optimizer/logical/Sort.hpp

```diff
@@ -137,6 +150,19 @@ class Sort : public Logical {
     addChild(input_);
   }
 
+  Sort(const LogicalPtr &input,
+       const std::vector<expressions::AttributeReferencePtr> &&sort_attributes,
+       const std::vector<bool> &&sort_ascending,
+       const std::vector<bool> &&nulls_first_flags,
+       const int limit)
+      : input_(input),
+        sort_attributes_(std::move(sort_attributes)),
+        sort_ascending_(std::move(sort_ascending)),
+        nulls_first_flags_(std::move(nulls_first_flags)),
+        limit_(limit) {
+    addChild(input_);
+  }
```

Comment:
Remove `const` for all rvalue reference (`&&`) arguments, otherwise it is not really doing the move construction.